bin/install-dependencies.sh

```diff
@@ -3,17 +3,14 @@
 pip=${1-'/usr/bin/env pip'}
 
 ckan_harvest_fork='alphagov'
-ckan_harvest_sha='5507adc82585a273c51eaebdf7e1d3bbe7d0f9dc'
+ckan_harvest_sha='7b251f0e3b26e9567025b833228676b8801979d0'
 
 ckan_dcat_fork='ckan'
-ckan_dcat_sha='8c55890e7bf9644460ae71470f37abbd9295c9fa'
+ckan_dcat_sha='b757e5be643a17f08b1bb102348c370abee149d5'
 
 ckan_spatial_fork='alphagov'
 ckan_spatial_sha='c35e1249cfb3c7b2f1286f433fd51c6fe8f05b72'
 
-ckan_s3_resources_fork='alphagov'
-ckan_s3_resources_sha='81eb36fb51da5e216e9405a7ad64c4096881ca85'
-
 ckan_fork='ckan'
```

Comment:
If we really want to tidy up and stop maintaining so much redundant custom code, we can remove this line and replace `ckan_fork` with `ckan` in the URL on lines 30-32.